### ISSUE TITLE
1.11.10 SCORM not loading

### ISSUE THREAD

**Issue by lonesomewalker** [chamilo/chamilo-lms#2946]
**Describe**
Simple SCORM course can't load.
Error log says:
PHP Warning:  Declaration of UniqueAnswer::return_header($exercise, $counter = NULL, $score = NULL) should be compatible with Question::return_header(Exercise $exercise, $counter = NULL, $score = Array) in /var/www/chamilo/main/exercise/unique_answer.class.php on line 19, referer: http://192.168.43.139/courses/GYFKURS001/scorm/Musterkurs/assets/htm/blank.htm

SCORM Cloud says: package 100% valid

**To Reproduce**
https://11.chamilo.org/courses/MOB0001/index.php
and go to learning path

**Additional context**
zipped SCORM package.
[Musterkurs.zip](https://github.com/chamilo/chamilo-lms/files/3358867/Musterkurs.zip)

**Comment by jmontoyaa** [comment]
Warning because this change: 

https://github.com/chamilo/chamilo-lms/commit/2c588909ef43949e635c9468b7ef19a251485f41

**Comment by lonesomewalker** [comment]
No, not only the warning is relevant, also the extraction of the scorm package.
After extraction, you should still have folders and files, instead we get flat file -> all in one folder.

**Comment by jmontoyaa** [comment]
There's a problem with your zip package, when I unzip the file, I have all files in one folder, just using ubuntu ark zip manager:

![Screenshot_20190704_141805](https://user-images.githubusercontent.com/158935/60666128-9bd82f80-9e66-11e9-994c-6c0dbd94b050.png)

**Comment by lonesomewalker** [comment]
Feel free to extract on the commandline with unzip.
File is a valid zip file, also the structure in it.
Working in several other LMS like Moodle and ILIAS.

**Comment by Coursenligne** [comment]
I had an issue uploading SCORM files, I changed the maximum upload size in php.ini file (using LAMP or WAMP) and all seems to work so far.
I hope this helps.

**Comment by opensourcecompany** [comment]
Upload size is 512M, should not be an issue ;-)

**Comment by lonesomewalker** [comment]
Any update on this?
Maybe somebody can give me a hint where the extraction of the uploaded SCORM package happens, so i will investigate further this on my own.

**Comment by ywarnier** [comment]
As @jmontoyaa mentioned above, there seems to be something wrong with the zip format itself (backslashes instead of slashes, which makes the unzip in Chamilo interpret them as all files in the same root folder, with strange "\" signs in their names). Not sure how the scorm cloud and other LMSes accept it. Maybe they have special settings to support this type of Windows Zip...
I've never seen one react like this before (even for ZIPs created under Windows).
